Project1/Unit.cpp

```diff
@@ -45,6 +45,15 @@ Combat& Unit::getCombat() {
 	return _combat;
 }
 
+void Unit::refresh() {
+	//TODO(T or W): Code start of turn stuff.
+}
+void Unit::newTurn() {
+	//TODO(T or W): code start of turn stuff.
+}
+bool Unit::isTired() const {
+	return true; //TODO(Weston): Should this be in a component like isDead() ? 
```

Comment:
Not in with isDead, but we should put all the level/turn info together.
Has attacked? and Movement remaining are the ones I can think of off the top of my head, but I vaguely remember wanting other ones too.